runtimes/typescript/lbr-prelude/src/LambdaBuffers/Runtime/Prelude.ts

```diff
@@ -307,7 +307,7 @@ export const jsonText: Json<Text> = {
 export type Maybe<A> = A | undefined;
 
 /**
- * `Eq` instance for `Maybe<A>`
+ * {@link Eq} instance for {@link Maybe}
```

Comment:
Why don't you do the same thing you did with Either? Would rather we stick to a pattern for representing enums rather than flerting with undefined